### PR TITLE
feat(ButtonLink): allow disabled state

### PR DESCRIPTION
**Pull request by atabel**
https://jira.tid.es/browse/WEB-518

### CODE REVIEW

**Review by yceballost — APPROVED** [review]

**Comment by yceballost** [comment]
Please, change [button story](https://mistica-jfp62848j-tuentisre.vercel.app/?path=/story/components-touchables-button--type-of-buttons) to add disabled behavior to ButtonLink

**Review by pladaria — APPROVED** [review]

File: src/button.tsx [hunk 1]
```diff
@@ -517,6 +519,7 @@ export const ButtonLink = React.forwardRef<
                 {props.children}
             </Text2>
         ),
+        disabled: props.disabled,
```
Comment (pladaria) [inline src/button.tsx:522]: You should also read the form state and if sending, automatically disable the button.

See the button implementation

Comment (atabel) [reply src/button.tsx:522]: I see we are looking for form status in TextLinks too, so it makes sense to be consistent here

**Comment by atabel** [comment]
> Please, change [button story](https://mistica-jfp62848j-tuentisre.vercel.app/?path=/story/components-touchables-button--type-of-buttons) to add disabled behavior to ButtonLink

done

File: src/button.tsx (hunk 1 above)
Comment (atabel) [reply src/button.tsx]: done

**Review by pladaria — APPROVED** [review]

**Pull request merged**